scripts/kind-e2e/e2e.sh

```diff
@@ -240,3 +240,14 @@ setup_broker
 setup_cluster2_gateway
 setup_cluster3_gateway
 test_connection
+
+if [[ $1 = keep ]]; then
+    echo "your 3 virtual clusters are deployed and working properly with your local"
+    echo "submariner source code, and can be accessed with:"
+    echo ""
+    echo "export KUBECONFIG=\$(echo \$(pwd)/output/kind-config/local-dev/kind-config-cluster{1..3} | sed 's/ /:/g')"
```

Comment:
It’s a shame we can’t easily calculate the real value of `KUBECONFIG` for the user...

Reply:
in case not in root dir, the KUBECONFIG won't setup correctly. Run this instead:
`export KUBECONFIG=$(echo $(git rev-parse --show-toplevel)/output/kind-config/local-dev/kind-config-cluster{1..3} | sed 's/ /:/g')`

Reply:
Done, thanks! :)